frontend/src/__tests__/cypress/cypress/tests/e2e/settings/servingRuntimes/testMultiModelServingRuntimeCreation.cy.ts

```diff
@@ -68,8 +68,8 @@ describe('Verify Admins Can Import and Delete a Custom Multi-Model Serving Runti
       .find()
       .within(() => {
         servingRuntimes.findEditModel().click();
-        servingRuntimes.findDeleteModel().click();
       });
+    servingRuntimes.findDeleteModel().click();
```

Comment:
Please also make the same change to the _testSingleServingRuntimeCreation.cy_ test which is currently failing.

Reply:
Fixed in [c7a3daf](https://github.com/opendatahub-io/odh-dashboard/pull/3549/commits/c7a3daffb15eff5383f55f7a7670f3e319b35669)